spacy/util.py

```diff
@@ -467,7 +467,10 @@ def make_tempdir():
     """
     d = Path(tempfile.mkdtemp())
     yield d
-    shutil.rmtree(str(d))
+    try:
+        shutil.rmtree(str(d))
+    except PermissionError as e:
```

Comment:
This is strange indeed... Maybe there's something we can set here? Or maybe it's just how Windows works? I vaguely remember something like this coming up in our test suite when working with temp files and directories. I wonder how we fixed that... or did we just skip those on Windows? 🤪

Reply:
I don't remember recently running into issues with temp files on Windows for the tests. We use it all the time. This particular issue seems to be triggered by a specific file that is "open" or something according to Windows. Dunno.